app/models/subscription.rb

```diff
@@ -1,7 +1,11 @@
 class Subscription < ActiveRecord::Base
+  attr_accessor :donator_cpf
+
   validates_presence_of :status, :payment_method, :charging_day,
                         :plan_id, :user_id, :project_id
 
+  validates :donator_cpf, presence: true, on: :create
```

Comment:
I think you can apply the same syntax here: `validates_presence_of : donator_cpf, on: :create`